app/src/main/java/com/willowtree/vocable/customviews/VocableImageButton.kt

```diff
@@ -29,17 +29,19 @@ class VocableImageButton @JvmOverloads constructor(
     }
 
     override fun onPointerEnter() {
-        buttonJob = backgroundScope.launch {
-            uiScope.launch {
-                isSelected = true
-            }
-
-            delay(DEFAULT_TTS_TIMEOUT)
-
-            uiScope.launch {
-                isSelected = false
-                isPressed = true
-                performAction()
+        if (this.isEnabled) {
```

Comment:
I added a check to line 170 of `BaseActivity` so it won't recognize the view intersect if it's disabled. So you might not need this here.

Reply:
so we added this because head tracking was still happening even if it was disabled

Reply:
ah! we didn't have that change in, but josh rebased and got it so we removed this check 👍